edgesite/pkg/controller/constants/service.go

```diff
@@ -0,0 +1,32 @@
+package constants
+
+// Service level constants
+const (
+	// module
+	EdgeControllerModuleName     = "controller"
+	CloudHubControllerModuleName = "cloudhub"
+
+	// Resource sep
+	ResourceSep               = "/"
+	ResourceSliceLength       = 5
+	ResourceSliceLengthQuery  = 4
+	ResourceNodeIndex         = 0
+        ResourceNodeIDIndex       = 1
+	ResourceNamespaceIndex    = 0
+	ResourceResourceTypeIndex = 1
+	ResourceResourceNameIndex = 2
+	ResourceNode              = "node"
+
+	// Group
+	GroupResource = "resource"
+
+	// Nvidia Constants
+	// NvidiaGPUStatusAnnotationKey is the key of the node annotation for GPU status
+	NvidiaGPUStatusAnnotationKey = "huawei.com/gpu-status"
```

Comment:
Please replace `huawei.com` with `gpu.kubeedge.io`or something else, it has nothing to do with the vendor here.